hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2506,6 +2506,14 @@
       for Ozone Manager Admin protocol.
     </description>
   </property>
+  <property>
+    <name>ozone.om.transport.class</name>
+    <value>org.apache.hadoop.ozone.om.protocolPB.GrpcOmTransportFactory</value>
```

Comment:
So, do we need to make [the actual change](https://github.com/apache/ozone/blob/master/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java#L341-L341) to make `GrpcOmTransportFactory` the default value for `ozone.om.transport.class`.

Reply:
No, `ozone-default.xml` supercedes the `OMConfigKeys` default value. The tests get the transport class from `OMConfigKeys`, so this way hadoop config will continue getting used there.